File: examples/csharp/SubmitTask.cs
using System;
using System.Net.Http;
using System.Net.Http.Json;
using System.Threading.Tasks;

public class SubmitTask
{
    public static async Task Main(string[] args)
    {
        var client = new HttpClient { BaseAddress = new Uri("http://localhost:5000") };
        var payload = new { AgentType = "dummy", Prompt = "Reverse the string 'hello'" };
        var resp = await client.PostAsJsonAsync("/tasks", payload);
        resp.EnsureSuccessStatusCode();
        var body = await resp.Content.ReadFromJsonAsync<object>();
        Console.WriteLine("Submitted: " + System.Text.Json.JsonSerializer.Serialize(body));
    }
}

File: examples/python/submit_task.py
import requests
import sys

def submit(prompt):
    payload = {"AgentType": "dummy", "Prompt": prompt}
    r = requests.post("http://localhost:5156/tasks", json=payload)
    r.raise_for_status()
    print("Submitted:", r.json())

if __name__ == '__main__':
    p = sys.argv[1] if len(sys.argv) > 1 else "Echo this"
    submit(p)
